.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Bug report
+about: Create a report to help us improve Zowe CLI
+labels: bug
+---
+
+<!-- Thanks for deciding to open an issue. Before submitting, please see the following information. -->
+
+<!-- Before opening a new issue, please search our existing issues: https://github.com/zowe/zowe-cli/issues -->
+
+<!--Also see the Troubleshooting section in the Zowe Docs website: https://docs.zowe.org/stable/troubleshoot/cli/known-cli
+-->
+
+**Describe your environment**
+
+<!--
+- Zowe CLI version installed (run command `zowe --version`):
+- Installed plug-ins and their version numbers (run command `zowe plugins list --short`):
+- Node.js and NPM versions installed (run `node --version`, `npm --version`):
+- Environment variables in use:
+- Operating system and version:
+- Shell/terminal (bash, cmd, powershell, etc...):
+- Daemon mode enabled or disabled:
+- For more information, see Gathering information to troubleshoot Zowe CLI
+-->
+
+**Describe the bug**
+
+<!-- A clear and concise description of the bug or error. -->
+
+
+**Expected and actual results**
+
+<!--
+Details about the behavior:
```

Comment:
Yeah, I think it might be better for people to focus more on describing their problem and the expected results.
And then they can think about their environment details and any additional context that may help 😋

Reply:
I'll edit accordingly. thank ya!